sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_models.py

```diff
@@ -2258,6 +2258,7 @@ def to_dict(self):
             if self.fields
             else {},
             "confidence": self.confidence,
+            "_parent": self._parent,
```

Comment:
Thoughts on special processing for this? At the moment this works with the get_words/get_lines calls.

Reply:
This is setting the generated model to the `_parent` key, though, so not converting the whole response to_dict as promised.

Check with Anna, but I don't think we want to include the _parent in the to_dict/from_dict, similar to how Will was talking about how the information wouldn't be round-tripped through JSON. This adds a lot to the dict, and it's no longer just a ref once we fully convert the parent to a dict :(

Reply:
Yeah agree, will talk to Anna and see what she things :/